model/workflow.go

```diff
@@ -19,6 +19,17 @@ import (
 	"fmt"
 )
 
+// InvokeKind defines how the target is invoked.
+type InvokeKind string
+
+const (
```

Comment:
I don't think we need to break constants in different blocks.

Reply:
That's fine since we keep the domain separated.